### PR TITLE
Wj branch

### PR DESCRIPTION
**Pull request by wjlim-14**
add: create pipeline folder

### CODE REVIEW

File: pipeline/wj_luigi.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+        -------------------------------------------
+        Summary (First 5 rows order by pickup time)
+        -------------------------------------------
+        {'|'.join(df.columns.to_list())}
+        {newline.join(
+            list("|".join(map(
+                str, df.values.tolist()[:5][i])) 
+                for i in range(5)))}
+        '''
```
Comment (geraldooi) [inline pipeline/wj_luigi.py:133]: image
Please check your indentation, your email look like this.

**Review by geraldooi — CHANGES_REQUESTED** [review]
Try to solve the indentation issue before merging to the master branch.

File: pipeline/wj_luigi.py (hunk 1 above)
Comment (wjlim-14) [reply pipeline/wj_luigi.py:132]: Email message indentation issues fixed.
image

File: pipeline/wj_luigi.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        Average duration per trip (in minutes): {statistics.mean(df['trip_duration'])} \n
+
+        -------------------------------------------
+        Summary (First 5 rows order by pickup time)
+        -------------------------------------------
+        {'|'.join(df.columns.to_list())}
+        {newline.join(
+            list("|".join(map(
+                str, df.values.tolist()[:5][i])) 
+                for i in range(5)))}
```
Comment (geraldooi) [inline pipeline/wj_luigi.py:131]:
```suggestion
        message = dedent(f'''
        Dear All,

        I found something,

        From "Upper East Side South" -> "Upper East Side North": \n
        Number of rows selected: {df.shape[0]} \n
        Average passenger count per trip: {statistics.mean(df['passenger_count'])} \n
        Average total amount per trip: {statistics.mean(df['total_amount'])} \n
        Average distance per trip: {statistics.mean(df['trip_distance'])} \n
        Average duration per trip (in minutes): {statistics.mean(df['trip_duration'])} \n

        -------------------------------------------
        Summary (First 5 rows order by pickup time)
        -------------------------------------------
        {'|'.join(df.columns.to_list())}
        ''')
        for i in range(5):
            message += '|'.join(map(str, df.values.tolist()[:5][i])) + '\n'
```

**Review by geraldooi — CHANGES_REQUESTED** [review]

**Review by geraldooi — APPROVED** [review]

**Pull request merged**